### ISSUE TITLE
[Feature request] advanced tasker support

### ISSUE THREAD

**Issue by Perksls** [laurencedawson/reddit-sync-development#1640]
It'd be awesome if reddit sync exposed internal settings to tasker or autoapps, not just for sync, but imagine being able to toggle nsfw post when you're not home, toggle night mode when some conditions are met, change account, card view, toggle preload images to save mobile data etc

**Comment by laurencedawson** [comment]
It would have to be very specific options I think.

**Comment by Perksls** [comment]
yes i agree, but some of the exemples i gave above would be awesome. maybe in reddit thay would manage to give you more suggestions, but imo this would be one of the most awesome features

**Comment by Moleculor** [comment]
> 
> 
> It would have to be very specific options I think.

I don't think it necessarily would? As long as you can simply identify the option you want to change, and what you want to change it to, I think everything in the app could be switched via an intent or some other option. 

I'm eagerly waiting for this to be implemented, if only for the NSFW toggle so I can shut off NSFW stuff when necessary, so I don't accidentally forget.
